src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  display: inline-flex;
+  flex-flow: column;
```

Comment:
if you use inline tag like `span` in HTML instead of `div` then you do not require line 3 and 4